README.md

```diff
@@ -123,6 +123,15 @@ Docker is the easiest way to enable PyTorch/JAX GPU support on Linux since only
 
    </details>
 
+## Packages to be installed from source
+
+some of the dependencies need to be installed from source like `ctcdecode`, here's the steps for source installation of `ctcdecode`
+
+1) git clone https://github.com/parlance/ctcdecode.git
```

Comment:
also this needs to be `git clone --recursive https://github.com/parlance/ctcdecode.git` to get it to work correctly